front/app/containers/EventsPage/UpcomingEvents.tsx

```diff
@@ -13,8 +13,8 @@ const StyledEventsViewer = styled(EventsViewer)`
 
 export default injectIntl<InjectedIntlProps>(({ intl }) => (
   <StyledEventsViewer
-    title={intl.formatMessage(messages.upcomingEvents)}
-    fallbackMessage={messages.noUpcomingEvents}
+    title={intl.formatMessage(messages.upcomingAndOngoingEvents)}
```

Comment:
Thanks for taking following through and improving the naming here too. :+1: Makes the intention very clear.